data/raw/datapackage.json

```
@@ -95,7 +77,7 @@
                         "long_name": "Sex",
                         "nombre_largo": "Sexo",
                         "type": "string",
-                        "description": "Sexo del Ave Laysan extraído por medio sanguineo donde H representa a hembras y M a machos",
+                        "description": "Sexo del albatros de Laysan determinado por medio sanguineo. Nomenclatura: H-Hembras y M-Machos.",

```

Aquí no estoy segura que la palabra _método_ sea la adecuada, pero _medio_ se oye raro.
```suggestion
                        "description": "Sexo del albatros de Laysan determinado por método sanguineo. Nomenclatura: H-Hembras y M-Machos.",

```